lib/routes/sdu/qd/xyb.ts

```diff
@@ -0,0 +1,88 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import { finishArticleItem } from '@/utils/wechat-mp';
+
+const host = 'https://xyb.qd.sdu.edu.cn/';
+
+const typeMap = {
+    gztz: {
+        title: '工作通知',
+        url: 'gztz.htm',
+    },
+};
+
+export const route: Route = {
+    path: '/qd/xyb/:type?',
+    categories: ['university'],
+    example: '/sdu/qd/xyb/gztz',
+    parameters: { type: '默认为`gztz`' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '青岛校区学科建设与研究生教育办公室',
+    maintainers: ['kukeya'],
+    handler,
+    description: `| 工作通知 | 
+  | -------- |
+  | gztz     | `,
+};
+
+async function handler(ctx) {
+    const type = ctx.req.param('type') ?? 'gztz';
+    const limit = ctx.req.param('limit') ?? 10;
+
+    const link = new URL(typeMap[type].url, host).href;
+    const response = await got(link);
+    let pageText: string, pageTextShort: string;
+
+    const $ = load(response.data);
+
+    let item = $('.list li')
+        .toArray()
+        .map((e) => {
+            e = $(e);
+            const a = e.find('a');
+            return {
+                title: a.text().slice(1).trim(),
+                link: a.attr('href').startsWith('info/') ? host + a.attr('href') : a.attr('href'),
+                pubDate: parseDate(e.find('b').text().trim(), 'YYYY-MM-DD'),
+            };
+        })
+        .slice(0, limit);
+
+    item = await Promise.all(
+        item.map((item) =>
+            cache.tryGet(item.link, async () => {
+                if (new URL(item.link).hostname === 'mp.weixin.qq.com') {
+                    return finishArticleItem(item);
+                }
+                const response = await got(item.link);
+                const $ = load(response.data);
+
+                pageText = $('.v_news_content').text().trim().slice(0, 100);
+                if (pageText.length >= 100) {
+                    pageTextShort = pageText.slice(0, 100);
+                    pageTextShort += '...';
+                }
+                item.description = pageTextShort;
```

Comment:
RSSHub will trim the description if necessary
https://github.com/DIYgod/RSSHub/blob/7ad41f89c07328f7b91018ae647824eed62edcc0/lib/middleware/parameter.ts#L402-L410

Do not trim it yourself.